health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/bill_payment_details.js

```
@@ -0,0 +1,732 @@
+
+    const rateBreakup = rateObj?.rateBreakup || {};
+    const wage =
+      (rateBreakup.FOOD || 0) +
+      (rateBreakup.TRAVEL || 0) +
+      (rateBreakup.PER_DAY || 0);
+      return {
+        ...billDetail,
+        givenName: individual?.name?.givenName,
+        mobileNumber: individual?.mobileNumber,
+        userId: individual?.userDetails?.username,
+        wage: wage+" "+workerRatesData?.currency,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)